Samples/RadarIO.Maui.Sample/MainPage.xaml.cs

```diff
@@ -0,0 +1,32 @@
+﻿using static RadarIO.Sample.Demo;
+
+namespace RadarIO.Maui.Sample;
+
+public partial class MainPage : ContentPage
+{
+    public MainPage()
+    {
+        InitializeComponent();
+        Initialize(new RadarSDKImpl());
```

Comment:
Our Xamarin documentation looks like this: https://radar.com/documentation/sdk/xamarin#initialize

Does this code need to do:
```
using RadarIO.Maui;
using static RadarIO.Maui.RadarSingleton;
``` 

Do we need to leverage the singleton anymore?